frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataBMC.js

```diff
@@ -206,14 +206,14 @@ const controlDataBMC = [
             required: true,
         },
     },
-    {
-        name: 'cluster.create.ocp.singleNode',
-        tooltip: 'tooltip.cluster.create.ocp.singleNode',
-        id: 'singleNode',
-        type: 'checkbox',
-        active: false,
-        hidden: isHidden_lt_OCP48,
-    },
+    //    {
+    //        name: 'cluster.create.ocp.singleNode',
+    //        tooltip: 'tooltip.cluster.create.ocp.singleNode',
+    //        id: 'singleNode',
+    //        type: 'checkbox',
+    //        active: false,
+    //        hidden: isHidden_lt_OCP48,
+    //    },
```

Comment:
Since it will never be supported, I guess we could just delete this stuff

Reply:
True. Deleted.